password.c

```diff
@@ -1,3 +1,4 @@
+#define _BSD_SOURCE // for explicit_bzero
```

Comment:
Defining feature tests macro in the source file is somewhat unideal, only doing it here because that's what is also done elsewhere in the code. Refer to [this post](https://ariadne.space/2021/12/21/stop-defining-feature-test-macros-in-your-code/) for more information.